docs/interpreter/python.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: "Python Interpreter"
+description: "Python Interpreter"
+group: manual
+---
+{% include JB/setup %}
+
+## Python 2 & 3 Interpreter for Apache Zeppelin
+
+## Configuration
+<table class="table-configuration">
+  <tr>
+    <th>Property</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>python</td>
+    <td>python</td>
+    <td>Path of the already installed Python binary (could be python2 or python3).
+    If python is not in your $PATH you can set the absolute directory (example : /usr/bin/python)
+    </td>
+  </tr>
+</table>
+
+## Enabling Python Interpreter
+
+In a notebook, to enable the **Python** interpreter, click on the **Gear** icon and select **Python**
+
+## Using the Python Interpreter
+
+In a paragraph, use **_%python_** to select the **Python** interpreter and then input all commands.
+
+The interpreter can only work if you already have python installed (the interpreter doesn't bring it own python binaries).
+
+To access the help, type **help()**
+
+## Python modules
+The interpreter can use all modules already installed (with pip, easy_install...)
+
+## Apply Zeppelin Dynamic Forms
+You can leverage [Zeppelin Dynamic Form]({{BASE_PATH}}/manual/dynamicform.html) inside your Python code.
+Example : 
+```bash
+%python
+# Input fom
+print (z.input("f1","defaultValue"))
+
+# Select fom
+print (z.select("f1",[("o1","1"),("o2","2")],"2"))
+
+#Checkbox form
+print("".join(z.checkbox("f3", [("o1","1"), ("o2","2")],["1"])))
+```
+
+## Zeppelin features not fully supported by the Python Interpreter
+
+* Interrupt a paragraph execution (`cancel()` method) is currently only supported in Linux and MacOs. If interpreter runs in another operating system (for instance MS Windows) , interrupt a paragraph will close the whole interpreter.
+* Progression bar in webUI  (`getProgress()` method) is currently not implemented.
+* Code-completion is currently not implemented.
+* Zeppelin Dynamic Form can only be used if py4j Python library is installed in your system. If not, you can install it with `pip install py4j`.
```

Comment:
We just check py4j when we start the interpreter.

If py4j is not available, the interpreter will output "you must install py4j Python module (pip install py4j) to use Zeppelin dynamic forms features" if you try to use dynamic form (see the end of bootstrap.py to get more details about the implementation)